idl/raft_storage.idl.hh

```diff
@@ -33,7 +33,7 @@ struct server_address {
 
 struct config_member {
     raft::server_address addr;
-    bool can_vote;
+    raft::can_vote_t can_vote;
 };
```

Comment:
Scylla’s approach to commits is to make them self-contained, explanatory chunks of changes with clear, descriptive messages. This simplifies both the review process and later maintenance, such as triaging user issues. Code changes resulting from PR comments should be incorporated into the corresponding commits using `git rebase -i` (or equivalent IDE tools).